docs/introduction/what-is-tradetrust.md

```diff
@@ -4,11 +4,25 @@ title: What is TradeTrust?
 sidebar_label: What is TradeTrust
 ---
 
-TradeTrust is a set of Governance & Legal Frameworks, Standards and future-ready Digital Infrastructure, all of which facilitates the interoperability of electronic trade documents exchanged between different digital ecosystems.
+TradeTrust is a framework that comprises globally-accepted standards that connect governments and businesses to a public blockchain to enable trusted interoperability of electronic trade documents across digital platforms AND it is offered as a digital utility.
```

Comment:
the AND with caps is requested by the client?

Reply:
as per written in the pivotal